app/src/helpers/requests/uploadDocument.test.ts

```diff
@@ -37,6 +43,27 @@ describe('[POST] updateDocumentState', () => {
     });
 });
 
+describe('uploadDocumentToS3', () => {
```

Comment:
note: I added this test because sonarcloud marked this function as untested and that affected the coverage %.